.rubocop.yml

```diff
@@ -127,7 +142,6 @@ Lint/UselessTimes:
 Lint/Void:
   Exclude:
     - lib/jekyll/site.rb
-
```

Comment:
Can we have these separators between Cop departments back? Or will they get removed automatically with autocorrect again?

Reply:
YAMLint 🤷🏻